.travis.yml

```diff
@@ -1,6 +1,7 @@
 dist: focal
 language: node_js
 node_js:
+    - lts/*
```

Comment:
Please tell me the reason why this line was added.

Reply:
It was added because of a suggestion from jeonghee over a conversation on slack. I see that other repos also has it and I thought it was ok to update this project as well.
Do you think is not neccessary?

Reply:
Thank you for the background explanation. Let's fix it.